File: 1791.find-center-of-star-graph.java
/*
 * @lc app=leetcode id=1791 lang=java
 *
 * [1791] Find Center of Star Graph
 */

// @lc code=start
class Solution {
    public int findCenter(int[][] edges) {
      // we get two elements in each sub array
      // whatever element matches the second sub array that is the center
      if(edges[0][0]==edges[1][0]|| edges[0][0]==edges[1][1]){
          return edges[0][0];
      }
      return edges[0][1];
    }
}
// @lc code=end

File: 1791.find-center-of-star-graph.py
#
# @lc app=leetcode id=1791 lang=python3
#
# [1791] Find Center of Star Graph
#

# @lc code=start
from typing import List


class Solution:

                
    
        def findCenter(self, edges: List[List[int]]) -> int:
            n = len(edges)+1
            count = [0 for _ in range(n+1)]
            for relation in edges:
                count[relation[0]]+=1
                count[relation[1]]+=1
            return count.index(n-1)
            
            

        
# @lc code=end

